### DISCUSSION TITLE
Wrong license listed on NuGet

### DISCUSSION THREAD

**Discussion by kelemvor4** [QuestPDF/QuestPDF#107] [General]
FYI, the license url in NuGet points to the license for skiasharp instead of questpdf.

**Comment by MarcinZiabek** [comment]
I don't see anything wrong on the Nuget page. Please elaborate more on where the issue is exactly 😁

**Comment by kelemvor4** [comment]
I guess I'm going to have to eat my words.  I was certain that I clicked the "license" link in nuget (from within VS 2019), which sent me to skiasharp's license page.   However, now when I inspect or click that link it simply shows the main URL for MIT licenses.    

Odd, because that was the first time I'd heard of skiasharp.

I must be losing my mind, sorry to have wasted your time.

**Comment by MarcinZiabek** [comment]
No worries 😁 I'm glad that everything is in tack in terms of the license! Closing

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
